### ISSUE TITLE
Different focus borders used in menubar

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/text#2732]
Some of the menu icons use a primary color outline while most of them use the text colored one:

https://user-images.githubusercontent.com/3404133/179910654-f5cbfecd-6be5-4891-a354-a803b1852835.mov

**Comment by susnux** [comment]
Or like the emoji picker: No outline.
For me it looks even weirder (Firefox 102):
![focused items are filled with text color](https://user-images.githubusercontent.com/1855448/179950867-bd0531c9-9ba3-45f7-bb56-cb4c1afbe5bf.gif)

**Comment by juliushaertl** [comment]
The white background should be fixed with latest server https://github.com/nextcloud/server/pull/33246

**Comment by susnux** [comment]
> The white background should be fixed with latest server [nextcloud/server#33246](https://github.com/nextcloud/server/pull/33246)

@juliushaertl  Only if you have selected the dark theme, when using the system default theme and use a dark color theme, as I do, this still shows the white background.

**Comment by juliushaertl** [comment]
Weird, I'm using exactly the same constellation. The DarkTheme class is being pulled in depending on the prefers-color-scheme value. Maybe still in the browser cache?

**Comment by susnux** [comment]
I thought I refreshed the cache... but you are right.

Looks like Buttons use the text color, while ActionButtons (sub menus) use the primary color, and the emoji picker is special

**Comment by juliushaertl** [comment]
Related upstream issue [for button component](https://github.com/nextcloud/nextcloud-vue/issues/2824#issue-1301645695)

**Comment by juliushaertl** [comment]
This is fixed with 26.
